--- php54-pecl-mongodb-1.4.2/config.m4~	2018-03-06 22:50:22.000000000 +0200
+++ php54-pecl-mongodb-1.4.2/config.m4	2018-04-02 23:07:46.043110993 +0300
@@ -86,8 +86,8 @@
   PHP_MONGODB_FOUND_VERSION=`${PHP_CONFIG} --version`
   PHP_MONGODB_FOUND_VERNUM=`echo "${PHP_MONGODB_FOUND_VERSION}" | $AWK 'BEGIN { FS = "."; } { printf "%d", ([$]1 * 100 + [$]2) * 100 + [$]3;}'`
   AC_MSG_RESULT($PHP_MONGODB_FOUND_VERSION)
-  if test "$PHP_MONGODB_FOUND_VERNUM" -lt "50500"; then
-    AC_MSG_ERROR([not supported. Need a PHP version >= 5.5.0 (found $PHP_MONGODB_FOUND_VERSION)])
+  if test "$PHP_MONGODB_FOUND_VERNUM" -lt "50400"; then
+    AC_MSG_ERROR([not supported. Need a PHP version >= 5.4.0 (found $PHP_MONGODB_FOUND_VERSION)])
   fi
 
   PHP_ARG_ENABLE([developer-flags],
