--- php70-pecl-mongodb-1.2.11/tests/manager/manager-ctor-read_preference-004.phpt	2017-10-17 20:40:05.082945968 +0300
+++ php70-pecl-mongodb-1.2.11/tests/manager/manager-ctor-read_preference-004.phpt	2017-10-17 20:45:44.540046995 +0300
@@ -22,7 +22,6 @@
     // Alternative values for true in URI string (all other strings are false)
     ['mongodb://127.0.0.1/?slaveok=t', []],
     ['mongodb://127.0.0.1/?slaveok=1', []],
-    ['mongodb://127.0.0.1/?slaveok=anything_other_than_true_t_and_0', []],
     // Case insensitivity for URI string and array options
     ['mongodb://127.0.0.1/?slaveOk=True', []],
     ['mongodb://127.0.0.1/?SLAVEOK=TRUE', []],
@@ -101,10 +100,6 @@
 }
 object(MongoDB\Driver\ReadPreference)#%d (%d) {
   ["mode"]=>
-  string(7) "primary"
-}
-object(MongoDB\Driver\ReadPreference)#%d (%d) {
-  ["mode"]=>
   string(18) "secondaryPreferred"
 }
 object(MongoDB\Driver\ReadPreference)#%d (%d) {
